### ISSUE TITLE
switch   nividia-amdgpu

### ISSUE THREAD

**Issue by pepebc** [dglt1/optimus-switch#30]
Work perfect for nvidia 
but is it possible to switch to amdgpu?

**Comment by squadramunter** [comment]
No this is only for Nvidia GPU's. I assume you have a AMD APU?

**Comment by pepebc** [comment]
Laptop Hybrid  GeForce RTX 3070 Mobile - AMD Cezanne
I work very well with optimus-manager for a long time but I have found a little better performance with optimus-switch

**Comment by squadramunter** [comment]
> Work perfect for nvidia but is it possible to switch to amdgpu?

So you have APU with integrated graphics and a external GPU in your notebook? Is that correct?

**Comment by pepebc** [comment]
Laptop with AMD Ryzen 9 5900HX  CPU with 2  integrated: Radeon Graphics and Nvidia GeForce RTX 3070

**Comment by pepebc** [comment]
With optimus-switch when I change to Nvidia it works perfectly, with 3D rendering and photos it works better than optimus-manager. But I would need to work in office mode with amd.
Sorry for my english

**Comment by squadramunter** [comment]
Ah no problem. Your English is fine. Sounds like there is a way but not with this solution. I think you can do this by using a script to let a program run with one GPU or another.

Maybe this video can help you? https://www.youtube.com/watch?v=hLv28GJra8U

**Comment by pepebc** [comment]
I work at Manjaro Mate. I try to get the best possible performance when working with 3D, video and photos. I liked optimus-switch because I was surprised by Nvidia's performance. But there are days when I do office work and I prefer not to force the machine.
Thank you for your work

**Comment by squadramunter** [comment]
Distro's don't matter. All it's different is the package mananger. That makes a distrubution different of each other. If you use Ubuntu you use APT when using Arch you use pacman. etc. Manjaro is Arch but the way the drivers works and how to set something up on each Linux system is the same.

**Comment by squadramunter** [comment]
Don't be afraid to mess with the Linux Command line and if I may give you a tip. Use Timeshift. I think Manjaro does have timeshift included by default. So make sure you make a snapshot before messing around so you can do a roll-back when something breaks.

**Comment by pepebc** [comment]
I'm not afraid, I started when there were no graphical environments.
I've benchmarked Ubuntu Mate and Manjaro Mate, and there are performance differences. After working with Debian and Ubuntu for years I am now very happy with these Arch Linux based distributions.
And yes, I always have a backup in case something goes wrong.
Very kind for your attention.
I'm going to try something.
I just wanted you to know that this switch is awesome.
Thank you
